src/System.Security.Cryptography.Cng/tests/AesCngTests.cs

```diff
@@ -0,0 +1,96 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class AesCngTests
+    {
+        private const int BlockSizeBytes = 16;
+
+        private static readonly CngAlgorithm s_cngAlgorithm = new CngAlgorithm("AES");
+
+        // The ephemeral key has already been validated by the AesCipherTests suite.
+        // Therefore we can use the ephemeral key to validate the persisted key.
+        [OuterLoop]
```

Comment:
It'd be worth a comment on why these are outerloop (similar to the reasoning in the PR description)